### DISCUSSION TITLE
Setting sash thickness for PanedWindow

### DISCUSSION THREAD

**Discussion by rmcmichael** [israel-dryer/ttkbootstrap#531] [Q&A] [answered]
I'm using a `PanedWindow` for a two column layout.  The sash width is very thin, making it hard to grab with the mouse and resize the columns.  In reading the documentation for TCL/Tk and ttk, you can set a `sashthickness` property for the Sash style:

`style.configure("Sash", sashthickness=6)`

The documentation is not clear, but for plain TTK it looks like you can set the sash thickness for some styles, but not others.  I've found that the "clam" style lets me successfully set the sash thickness.

However, it is not working using ttkbootstrap.  Any ideas for how to set the sash thickness of a `PanedWindow` in ttkbootstrap?

Thanks

**Comment by israel-dryer** [comment, marked as answer]
You were on the right track — `style.configure("Sash", sashthickness=N)` **is** the correct call, and it does work in ttkbootstrap. The reason it looked like it didn't is **timing**.

ttkbootstrap builds widget styles lazily. If you configure `Sash` *before* the `PanedWindow` (or its style) has been built, ttkbootstrap's panedwindow style builder runs afterward and overwrites `sashthickness` back to its 2px default. I confirmed this: configuring before the widget → 2px; configuring **after** the widget exists → the value sticks.

So the fix is to set it after the widget is created:

```python
import ttkbootstrap as ttk

app = ttk.Window()
pw = ttk.Panedwindow(app, orient="horizontal")   # note: Panedwindow
pw.add(ttk.Frame(pw)); pw.add(ttk.Frame(pw))
pw.pack(fill="both", expand=True)

app.style.configure("Sash", sashthickness=8)      # AFTER the widget
```

One more gotcha: a `theme_use()` switch re-runs the builder and resets it again. If you change themes at runtime, re-apply on `<<ThemeChanged>>` (it fires after the rebuild, so it sticks):

```python
THICKNESS = 8
def _apply_sash(_=None):
    app.style.configure("Sash", sashthickness=THICKNESS)
_apply_sash()
app.bind("<<ThemeChanged>>", _apply_sash, "+")
```

Note `"Sash"` is a global ttk element — the thickness applies to every paned window in the app, and this is a Tk limitation (the sash thickness can't be scoped per-widget/per-style; the `ttk::panedwindow` C widget reads it from the fixed `Sash` style). One consistent thickness per app is really the intended model.

The rough edges here — having to know the `Sash` element name, and the theme-change reset — are tracked for 2.0 in #1161, where sash thickness will become a supported, theme-durable setting.
